src/doc/rustc/src/platform-support/armv7a-vex-v5.md

```diff
@@ -0,0 +1,91 @@
+# `armv7a-vex-v5`
+
+**Tier: 3**
+
+
+Allows compiling user programs for the [VEX V5 Brain](https://www.vexrobotics.com/276-4810.html), a microcontroller for educational and competitive robotics.
+
+Rust support for this target is not affiliated with VEX Robotics or IFI.
+
+## Target maintainers
+
+This target is maintained by members of the [vexide](https://github.com/vexide) organization:
+
+- [@Tropix126](https://github.com/Tropix126)
+- [@Gavin-Niederman](https://github.com/Gavin-Niederman)
+- [@max-niederman](https://github.com/max-niederman)
+- [@doinkythederp](https://github.com/doinkythederp)
+
+## Requirements
+
+This target is cross-compiled. Dynamic linking is unsupported.
+
+`#![no_std]` crates can be built using `build-std` to build `core` and optionally
+`alloc`. Unwinding panics are not yet supported.
+
+`std` is partially implemented, but many modules (such as `thread`, `process`, `net`, etc...) will return errors. An allocator is provided along with partial support for the `time`, `env` and `io` modules. Filesystem operations over SDCard through `std::fs` are partially supported within the restrictions of the user enviornment (e.g. directories cannot be created, filesystem objects cannot be removed).
+
+This target generates binaries in the ELF format that may uploaded to the brain with external tools.
+
+## Building the target
+
+Rust does not ship pre-compiled artifacts for this target. You can use the `build-std` feature to build ELF binaries with `std` support.
+
+`.cargo/config.toml`:
+
+```toml
+[build]
+target = "armv7a-vex-v5"
+
+[unstable]
+build-std = ["std", "panic_abort"]
+build-std-features = ["compiler-builtins-mem"]
+```
+
+## Building Rust programs
+
+The recommended way to build artifacts that run on V5 Brain is by using the [cargo-v5](https://github.com/vexide/cargo-v5) tool. This tool wraps the `cargo build` command by supplying arguments necessary to build the target, while also providing functionality for uploading over USB to a V5 Controller or Brain.
```

Comment:
We might not want to use the "recommended" phrasing because it implies an official recommendation by the Rust Project.